src/components/measureList/MeasureList.tsx

```diff
@@ -255,14 +253,23 @@ export default function MeasureList(props: {
     setOptionsOpen(false);
   };
 
-  const zipData = () => {
-    const zip = new JSzip();
-    zip.generateAsync({ type: "blob" }).then(function (content) {
-      saveAs(
-        content,
+  const zipData = async () => {
+    try {
+      const blobResponse = await measureServiceApi.getZippedMeasureData(
+        targetMeasure?.current?.id
+      );
+      const url = window.URL.createObjectURL(blobResponse);
+      const link = document.createElement("a");
+      link.href = url;
+      link.setAttribute(
+        "download",
         `${targetMeasure.current?.ecqmTitle}-v${targetMeasure.current?.version}-${targetMeasure.current?.model}.zip`
       );
-    });
+      document.body.appendChild(link);
+      link.click();
+    } catch (err) {
+      return err;
```

Comment:
should we do something with an error?

Reply:
nvm, seems out of scope.

Reply:
Yeah, this was out of scope so I that is the reason I am just returning the error